runtime/vm/compiler/compiler_pass.h

```diff
@@ -9,8 +9,6 @@
 #error "AOT runtime should not use compiler sources (including header files)"
 #endif  // defined(DART_PRECOMPILED_RUNTIME)
 
-#include <initializer_list>
```

Comment:
This file is also using `std::initializer_list` so this include has to stay.

Reply:
Yep. Per iwyu rule https://google.github.io/styleguide/cppguide.html#Include_What_You_Use

Reply:
Done.